packages/cozy-harvest-lib/src/components/KonnectorModal.jsx

```diff
@@ -10,17 +10,53 @@ import Modal, {
   ModalHeader
 } from 'cozy-ui/transpiled/react/Modal'
 import Spinner from 'cozy-ui/transpiled/react/Spinner'
-import { SubTitle } from 'cozy-ui/transpiled/react/Text'
 
 import accountMutations from '../connections/accounts'
 import triggersMutations from '../connections/triggers'
-import * as triggers from '../helpers/triggers'
+import * as triggersModel from '../helpers/triggers'
 import TriggerManager from './TriggerManager'
 import DeleteAccountCard from './cards/DeleteAccountCard'
 import LaunchTriggerCard from './cards/LaunchTriggerCard'
 import TriggerErrorInfo from './infos/TriggerErrorInfo'
 import withLocales from './hoc/withLocales'
+import SelectBox, {
+  reactSelectControl,
+  components
+} from 'cozy-ui/transpiled/react/SelectBox'
 
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import { Text } from 'cozy-ui/transpiled/react/Text'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+
+const MyAccountComponent = ({ name }) => {
```

Comment:
Need to split those into separate files.

I would also rename this one, maybe to `AccountSelectControl` ?